### PR TITLE
Add visibility of collector ingest rate and aggregator connection count

### PR DESCRIPTION
**Pull request by zyiou**

### CODE REVIEW

File: pkg/intermediate/aggregate.go [hunk 1]
```diff
@@ -128,6 +128,13 @@ func (a *AggregationProcess) Stop() {
 	a.stopChan <- true
 }
 
+// GetNumOfConnections returns total number of connections stored in map
+func (a *AggregationProcess) GetNumOfConnections() uint64 {
```
Comment (srikartati) [inline pkg/intermediate/aggregate.go:132]: Prefer `GetNumberOfFlows` as we use connections in exporter and collector.

**Review by srikartati — COMMENTED** [review]
Why not have a method exposing the number of clients connected to a server in the collector process?

File: pkg/intermediate/aggregate.go (hunk 1 above)
Comment (zyiou) [reply pkg/intermediate/aggregate.go]: Sure. I didn't see any. Thanks!

**Review by srikartati — DISMISSED** [review]
LGTM
Do you have a plan on how to consume these methods in Antrea? Logs/Prometheus metrics/antctl?

**Comment by zyiou** [comment]
> LGTM
> Do you have a plan on how to consume these methods in Antrea? Logs/Prometheus metrics/antctl?

I'm planning to add periodically logs in Antrea as discussed with @kais66 offline. PR is almost ready. We can add more support with antctl or prometheus metrics when we have bandwidth or upon users' request.

**Comment by srikartati** [comment]
> > LGTM
> > Do you have a plan on how to consume these methods in Antrea? Logs/Prometheus metrics/antctl?
> 
> I'm planning to add periodically logs in Antrea as discussed with @kais66 offline. PR is almost ready. We can add more support with antctl or prometheus metrics when we have bandwidth or upon users' request.

Using periodic logs sounds good.

**Comment by zyiou** [comment]
Rebased with main, need approval again. Thanks!

**Review by srikartati — APPROVED** [review]
lgtm

**Pull request merged**